test/fluree/db/ledger/general/todo_permissions.clj

```diff
@@ -127,7 +149,7 @@
     (is (= 5 (:block td-resp)))
 
     ;; there should be 4 tempids
```

Comment:
Note to self (more than anyone): remove comment because check is self-explanatory.  We can do that the next time we are updating tests.